tests/apps/pubsub-publisher/app.go

```diff
@@ -165,7 +450,11 @@ func performPublish(w http.ResponseWriter, r *http.Request) {
 }
 
 func performPublishHTTP(reqID string, topic string, jsonValue []byte, contentType string, metadata map[string]string) (int, error) {
-	url := fmt.Sprintf("http://localhost:%d/v1.0/publish/%s/%s", daprPortHTTP, pubsubName, topic)
+	psName := pubsubName
+	if strings.Contains(topic, bulkSubTopicIdentity) {
```

Comment:
e2e refactor: remove this and make use of `bulkPubsubMetaKey = "bulkPublishPubsubName"` that has been defined and extract it from metadata in the `performPublish` method and based on that pass on the pubsubName.

Reply:
Agreed, will do it separately under https://github.com/dapr/dapr/issues/5271. Added over there in the list. Resolving this comment here.